scripts/cleanup.py

```diff
@@ -1,5 +1,14 @@
+import re
 import pandas as pd
 
+def format_dial_codes(dial):
```

Comment:
@gradedSystem what was the old script that handled this case correctly?

Reply:
@anuveyatsu I believe the old script `scripts/statoids.py` should be handling this case:
https://github.com/datasets/country-codes/blob/92bbe1a42dd38503c43e29f8218d850748c29128/scripts/statoids.py#L82
however it doesn't seem to work in this case

Reply:
@gradedSystem ok so this is a unique case only for Dominican Republic. In this case I'm OK with the change you are proposing in this PR :+1:

Reply:
Yes it is the only case